app/extensions/string_extensions.rb

```diff
@@ -517,6 +517,45 @@ def gsub_html_special_chars
       html_safe # rubocop:disable Rails/OutputSafety
   end
 
+  # For integration test comparisons:
+  # Render special encoded characters as they appear in HTML
+  def render_html
```

Comment:
Would it be too annoying for me to request a more descriptive name? Without reading this method I would have no clue what "blah".render_html does.  Hmmm.  Okay, the really obvious choice ("unescape_html", because, well, that's _exactly_ what the method does) would also be inscrutable to me, most likely.  String and character encoding is always a pain in the butt no matter how you dress it up... :(

Reply:
Not too annoying. Yes — i considered `unescape_html` but i can never keep straight what escaping and unescaping mean, these are truly inscrutable words that seem to sometimes mean exactly the opposite of what i think. I chose `render_html` thinking more like, "render this html code `&8931;` in the page like it will be shown in the browser." 

That method was a pain in the butt to figure out, btw.

Reply:
I believe you.  Understood.